tests/bson/bson-utcdatetime-001.phpt

```diff
@@ -12,21 +12,21 @@ $manager = new MongoDB\Driver\Manager(STANDALONE);
 
 $classname = BSON_NAMESPACE . "\\UTCDateTime";
 $utcdatetime = new $classname("1416445411987");
-$result = $manager->executeInsert(NS, array('_id' => 1, 'x' => $utcdatetime));
-$query = new MongoDB\Driver\Query(array('_id' => 1));
-$cursor = $manager->executeQuery(NS, $query);
-$array = iterator_to_array($cursor);
```

Comment:
I isolated this change in https://github.com/jmikola/mongo-php-driver-prototype/commit/870ceb6a5f38453a22f2050dd8c4e5cfec4548d1. The only functional change was removing the `$result` assignment after the insert (since it was never used) and changing the DateTime `echo` to a `var_dump()` with a string cast.